using-timescaledb/continuous-aggregates.md

```diff
@@ -199,10 +199,18 @@ aggregate, such as views that are built on top of the continuous aggregate view.
 ### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
 When dropping data in a raw hypertable using the [`drop_chunks` function][api-drop-chunks]
 that has a continuous aggregate created on it, we must specify the `cascade_to_materializations`
-argument to the `drop_chunks` call. Currently, the only option for this argument is
-`true`, which will cause the continuous aggregate to drop all data associated
-with any chunks dropped from the raw hypertable. Further data retention options are
-planned for future releases (see [future work](#future-work)).
+argument to the `drop_chunks` call. A value of `true` will cause the continuous aggregate 
+to drop all data associated with any chunks dropped from the raw hypertable. A value
+of `false` will retain data in the continuous aggregate while dropping only the
+raw data.
+
+>:TIP: When dropping data from the raw hypertable while retaining data
```

Comment:
This tip should also be in the api docs as it is going to be quite confusing.

Reply:
Is this a blocker for merging into `master` @cevian ?